extension/src/experiments/model/tree.ts

```diff
@@ -238,17 +238,15 @@ export class ExperimentsTree
 
   private getExperimentIcon({
     displayColor,
-    status,
     type,
     selected
   }: {
     displayColor?: string
     label: string
-    status?: ExperimentStatus
     type?: ExperimentType
     selected?: boolean
   }): ThemeIcon | Uri | Resource {
-    if (isRunning(status)) {
```

Comment:
Are these helpers still useful if they're checking only the type? It was removed here, but used again in `extension/src/experiments/model/index.ts`. We should stay consistent and either use them everywhere or remove them.